src/ui/components/Avatar.js

```diff
@@ -1,20 +1,43 @@
-import React, { useState, useEffect } from "react";
+import React, { useEffect } from "react";
 import { useAuth0 } from "@auth0/auth0-react";
+import { gql, useMutation } from "@apollo/client";
 
 import { connect } from "react-redux";
 import { actions } from "ui/actions";
 import { getAvatarColor } from "ui/utils/user";
 import { setUserInBrowserPrefs } from "../utils/browser";
+import { ThreadFront } from "protocol/thread";
 
+const CREATE_SESSION = gql`
+  mutation AddSession($object: sessions_insert_input!) {
+    insert_sessions_one(object: $object) {
+      id
+      controller_id
+      recording_id
+    }
+  }
+`;
 const Avatar = props => {
   let { player, isFirstPlayer, updateUser } = props;
   let auth = useAuth0();
+  const [AddSession] = useMutation(CREATE_SESSION);
 
   useEffect(() => {
     setUserInBrowserPrefs(auth.user);
     updateUser(auth.user);
   }, [auth.user]);
 
+  useEffect(() => {
```

Comment:
Was there a specific reason for putting this in the Avatar component? I expected it to be higher in the tree, like `<Devtools />` maybe.

Reply:
That's a good call. i was look for auth data